fluid/vgg16.py

```diff
@@ -70,10 +70,16 @@ def conv_block(input, num_filter, groups, dropouts):
 def main():
     if args.data_set == "cifar10":
         classdim = 10
-        data_shape = [3, 32, 32]
+        if args.data_format == 'NCHW':
+            data_shape = [3, 32, 32]
+        else:
+            data_shape = [32, 32, 3]
     else:
         classdim = 102
-        data_shape = [3, 224, 224]
+        if args.data_format == 'NCHW':
+            data_shape = [3, 224, 224]
+        else:
+            data_shape = [224, 224, 3]
```

Comment:
The network is different for 224*224 input and 32*32 input, please refer to https://github.com/dzhwinter/benchmark/blob/master/tensorflow/resnet50.py#L360

Reply:
done.